Tests/TuistGeneratorTests/Extensions/XCTestCase+Extras.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+import XCTest
+
+extension XCTestCase {
```

Comment:
This extension is a good candidate to be in `TuisCoreTesting` *(made public)*

Reply:
Moved to `TuisCoreTesting`